dm/dm/master/server.go

```diff
@@ -2243,6 +2275,29 @@ func (s *Server) OperateRelay(ctx context.Context, req *pb.OperateRelayRequest)
 		return resp2, nil
 	}
 	resp2.Result = true
+	// TODO: now we make sure req.Source isn't empty and len(req.Worker)>=1 in dmctl,
+	// we need refactor the logic here if this behavior changed in the future
+	var workerToCheck []string
+	for _, worker := range req.Worker {
```

Comment:
`req.Worker` may empty after https://github.com/pingcap/ticdc/pull/3226 this merged 😅

maybe left a TODO here